assets/src/editor/events/dates-and-times/editor-date/grid-view/editor-date-entity-details.js

```diff
@@ -13,57 +13,58 @@ import {
 } from '@eventespresso/editor-hocs';
 import { routes } from '@eventespresso/eejs';
 import { __ } from '@eventespresso/i18n';
-import { dateTimeModel } from '@eventespresso/model';
-import { isModelEntityOfModel } from '@eventespresso/validators';
 
-const { MODEL_NAME: DATETIME } = dateTimeModel;
 const {
 	ADMIN_ROUTES,
 	ADMIN_ROUTE_ACTIONS,
 	ADMIN_ROUTE_ACTION_DEFAULT,
 	getAdminUrl,
 } = routes;
-/**
- * EditorDateEntityDetails
- *
- * @function
- * @param {Object} dateEntity    JSON object defining the Event Date
- * @return {string}    date details
- */
-class EditorDateEntityDetails extends Component {
+
+const EditorDateEntityDetails = ( {
+	eventEntity,
+	dateEntity,
+	venueEntity,
+	refreshed,
+	showDesc = 'excerpt',
+	showVenue = true,
+} ) => {
 	/**
 	 * dateName
 	 *
 	 * @function
 	 * @param {Object} dateEntity 	model object defining the Event Date
 	 * @return {string}    date name
 	 */
-	dateName = ( dateEntity ) => {
-		const htmlClass = classNames(
-			'ee-editor-date-name-heading',
-			{
-				'ee-long-title': dateEntity.name && dateEntity.name.length > 40,
-			}
-		);
-		return (
-			<h1 className={ htmlClass }>
-				<InlineEditInput
-					htmlId={ `event-date-name-${ dateEntity.id }` }
-					type="text"
-					value={ dateEntity.name }
-					onChange={ ( name ) => {
-						return this.updateName( name, dateEntity );
-					} }
-					label={ __( 'Date Name', 'event_espresso' ) }
-					noticeStyle={
-						{
-							left: '50px',
+	const dateName = useMemo(
+		() => {
+			const htmlClass = classNames(
+				'ee-editor-date-name-heading',
+				{
+					'ee-long-title': dateEntity.name && dateEntity.name.length > 40,
+				}
+			);
+			return (
+				<h1 className={ htmlClass }>
+					<InlineEditInput
+						htmlId={ `event-date-name-${ dateEntity.id }` }
+						type="text"
```

Comment:
noticed earlier that you were changing prop formatting like this to `type={ 'text' }` 
is that something we want to follow as a style guide?
I originally was just using `"` for string values, but have switched to braces and `'` for better consistency

Reply:
No real reason other than I get in a habit of using braces and phpStorm (for me at least) automatically adds them when it detects I'm doing a jsx element.